lib/private/Blurhash/Listener/GenerateBlurhashMetadata.php

```diff
@@ -64,7 +70,9 @@ public function handle(Event $event): void {
 			return;
 		}
 
-		$image = $this->resizedImageFromFile($file);
+		$preview = $this->preview->getPreview($file, 64, 64, cacheResult: false);
```

Comment:
The size was 30x30 before. That's what the `RESIZE_BOXSIZE` constant is used for.

```suggestion
		$preview = $this->preview->getPreview($file, self::RESIZE_BOXSIZE, self::RESIZE_BOXSIZE, cacheResult: false);
```

I think 64 would also be okay - but maybe we can drop the constant then / change it.

Reply:
https://github.com/nextcloud/server/pull/50494 has screenshots with 30px vs 300px.